lib/bundlex/project.ex

```diff
@@ -42,18 +43,22 @@ defmodule Bundlex.Project do
           src_base: String.t(),
           compiler_flags: [String.t()],
           linker_flags: [String.t()],
-          language: :c | :cpp
+          language: :c | :cpp,
+          interfaces: [:nif | :cnode]
```

Comment:
Setting this e.g to `interface: nif | :cnode` instead of `interfaces: [:nif | :cnode]` and writing in `bundlex.exs` in native config e.g. `interfaces: [:nif]` instead of `interface: :nif` still works. For simplicity we do not have to return `config_t` from `@callback project():: config_t`  declared here https://github.com/membraneframework/bundlex/blob/2bd83f0e5a393f1c91c89bbe65a7ddb844eaff9f/lib/bundlex/project.ex#L66 and everything works. Not sure if this is ok?

Reply:
Maybe keep it singular (`:interface`) and allow both atom and list of atoms. `Bunch.listify` may help with parsing.

> For simplicity we do not have to return config_t from @callback project():: config_t declared here

Can you elaborate?

Reply:
> Can you elaborate? 
Now it works, perhaps I missed something :roll_eyes: 
> Maybe keep it singular (:interface) and allow both atom and list of atoms. Bunch.listify may help with parsing.
Done